pkgs/development/libraries/gdal/default.nix

```
@@ -55,9 +55,9 @@ stdenv.mkDerivation rec {
   # - Unset CC and CXX as they confuse libtool.
   # - teach gdal that libdf is the legacy name for libhdf
   preConfigure = ''
-      substituteInPlace configure \
-      --replace "-lmfhdf -ldf" "-lmfhdf -lhdf"
-    '';
+    substituteInPlace configure \
+    --replace "-lmfhdf -ldf" "-lmfhdf -lhdf"
```

The new indentations (2 spaces) are better but line continuations should be indented as well.

```suggestion
      --replace "-lmfhdf -ldf" "-lmfhdf -lhdf"
```